src/inmanta/moduletool.py

```diff
@@ -178,20 +183,12 @@ def determine_new_version(self, old_version, version, major, minor, patch, dev):
                 elif len(opts) > 1:
                     LOGGER.error("You can use only one of the following options: --major, --minor or --patch")
                     return None
-                parts = old_version.base_version.split(".")
-                while len(parts) < 3:
-                    parts.append("0")
-                parts = [int(x) for x in parts]
-                if patch:
-                    parts[2] += 1
-                if minor:
-                    parts[1] += 1
-                    parts[2] = 0
-                if major:
-                    parts[0] += 1
-                    parts[1] = 0
-                    parts[2] = 0
-                outversion = ".".join([str(x) for x in parts])
+
+                change_type: Optional[ChangeType] = ChangeType.parse_from_bools(patch, minor, major)
```

Comment:
This is just the legacy method, right? I think it could be useful to add a comment at the method definitition stating that this is only used by the old `inmanta module commit` command.